pycontrails/models/cocip/contrail_properties.py

```diff
@@ -1213,22 +1214,22 @@ def plume_temporal_evolution(
     """
     # Convert dt to seconds value and use dtype of other variables
     dtype = np.result_type(width_t1, depth_t1, sigma_yz_t1, dsn_dz_t1, diffuse_h_t1, diffuse_v_t1)
-    dt_s = dt / np.timedelta64(1, "s")
-    dt_s = dt_s.astype(dtype, copy=False)
+    dt_s = units.dt_to_float_seconds(dt, dtype)
```

Comment:
Great! This seems like it should be `dt_to_seconds` since you are specifying the dtype here. I'll comment below in units.

Reply:
We should have added these utilities long ago.